src/pages/Timeline/Timeline.js

```diff
@@ -4,7 +4,6 @@ import CreatePost from "../../components/CreatePost";
 import Loading from "../../components/Loading";
 import Post from "../../components/Post";
 
-
 function Timeline() {
```

Comment:
Esse arquivo não deveria estar nessa branch pois não faz parte do que é descrito nesse commit. Se isso for para produção, uma feature incompleta e estática estará disponível para o cliente, o que não é o ideal e poderá causar conflitos no futuro.

Reply:
Nessa parte apenas uma linha foi apagada apenas correção minha pois fui testar o context estava sendo passado como terminei os testes ele acusou a mudança de 1 linha